art-feature.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Art Feature</title>
+    <link rel="icon" href="/img/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <h1 class="welcome-title">Your Art</h1>
+    <div class="editor">
+      <div class="html-container">
+        <input type="text" placeholder="Thinking is the enemy of creativity." />
+        <textarea id="html-code"></textarea>
```

Comment:
Textarea expands now both horizontally and vertically right... Should we maximize the width horizontally